src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+
+.catalog {
+  display: grid;
+  gap: 46px 48px;
+  box-sizing: border-box;
+  padding: 50px 40px;
+  justify-content: center;
+
+  font-family: Roboto, sans-serif;
```

Comment:
`font-family` should be set for `html` tag, so as not to duplicate this rule in other classes.